_maps/RandomRuins/LavaRuins/lavaland_biodome_beach.dmm

```diff
@@ -146,7 +146,7 @@
 /area/ruin/powered/beach)
 "aF" = (
 /obj/machinery/vending/boozeomat{
-	emagged = 1;
+	obj_flags = 5;
```

Comment:
I greatly dislike this just from an 'ability to understand what the edit is without looking it up' standpoint.

Reply:
Would you be alright with /emagged subtypes? There's only a few unique types involved here

Reply:
subtypes for the sake of mapping is just fine

Reply:
/obj/machinery/door/airlock/emagged
/obj/machinery/door/airlock/command/emagged
/obj/machinery/door/airlock/public/glass/emagged
/obj/machinery/door/airlock/research/emagged
/obj/machinery/vending/boozeomat/emagged
/obj/machinery/computer/arcade/battle/emagged
/obj/machinery/porta_turret/emagged
/mob/living/simple_animal/bot/secbot/beepsky/emagged

honestly this seems worse than just having an emagged = TRUE var tbh.

Reply:
beepsky isn't going to get one, changing it was an oversight